ocs_ci/utility/deployment_openshift_logging.py

```diff
@@ -0,0 +1,383 @@
+"""
+This module deploys the openshift-logging on the cluster
+EFK stack
+"""
+
+import logging
+import json
+
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.resources.pod import get_all_pods, get_pod_obj
+from ocs_ci.utility import templating
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility.retry import retry
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+def create_namespace(yaml_file):
+    """exec_cmd_on_pod(self, command, out_yaml_format=True):
+    Creation of namespace "openshift-operators-redhat"
+    for Elasticsearch-operator and "openshift-logging"
+    for ClusterLogging-operator
+
+    Args:
+        yaml_file (str) : Path to yaml file to create namespace
+
+    Returns:
+        namespace (Boolean): True if namespace is created and active, False
+        otherwise
+
+    Example:
+        create_namespace(yaml_file=constants.EO_NAMESPACE_YAML)
+    """
+
+    namespaces = ocp.OCP(kind=constants.NAMESPACES)
+
+    logger.info("Creating Namespaces")
+    namespaces.create(yaml_file=yaml_file)
+    namespace = namespaces.get()
+    for i in namespace:
```

Comment:
What does `i` represent? Let's give it a better name.

Reply:
Missed this. Will address in my next commit